### ISSUE TITLE
Accessible alt text

### ISSUE THREAD

**Issue by OliverWoolland** [UoMResearchIT/docker-introduction#2]
Moving to a more visual style of teaching means using more images, which should have properly written alt text to increase accessibility.

Should this be an immediate objective?
Does anyone know guidance on the style of writing for accessible alt text?
